crates/common/rlp/decode.rs

```diff
@@ -330,6 +336,34 @@ impl<T1: RLPDecode, T2: RLPDecode, T3: RLPDecode> RLPDecode for (T1, T2, T3) {
     }
 }
 
+impl<
```

Comment:
nit: a small comment stating that this decodes a 4-element list as a tuple would be helpful.

Reply:
Done!